mappings/net/minecraft/block/FenceBlock.mapping

```diff
@@ -4,5 +4,5 @@ CLASS net/minecraft/class_2354 net/minecraft/block/FenceBlock
 		ARG 1 state
 		ARG 2 neighborIsFullSquare
 		ARG 3 dir
-	METHOD method_26375 isFence (Lnet/minecraft/class_2680;)Z
+	METHOD method_26375 isCompatibleFence (Lnet/minecraft/class_2680;)Z
```

Comment:
I'm not a fan of `isCompatibleFence` in particular, but it is more correct than `isFence`, just vague. Perhaps:

```suggestion
	METHOD method_26375 canConnectToFence (Lnet/minecraft/class_2680;)Z
```